src/examples/zkapps/voting/demo.ts

```diff
@@ -56,12 +57,13 @@ let params: VotingAppParams = {
   votingKey,
   doProofs: true,
 };
+params.electionPreconditions.enforce = Bool(true);
```

Comment:
I had to slightly change the contracts preconditions to only enforce election time frames when wanted, otherwise the e2e on berkeley would be unnecessarily complex/longer take